### PR TITLE
GAE-storage v0.7.1-SNAPSHOT (with Spine v0.7.5-SNAPSHOT)

### PR DESCRIPTION
**Pull request by dmdashenkov**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,10 +1,11 @@
 project.ext {
-    SPINE_VERSION = '0.7.0';
-    PROTOBUF_VERSION = '3.1.0';
-    SLf4J_VERSION = "1.7.21";
-    PROTOBUF_DEPENDENCY = "com.google.protobuf:protoc:${project.PROTOBUF_VERSION}";
-    MAVEN_REPOSITORY_URL = 'http://maven.teamdev.com/repository/spine';
-    MAVEN_SNAPSHOT_REPOSITORY_URL = 'http://maven.teamdev.com/repository/spine-snapshots';
+    SPINE_VERSION = '0.7.5-SNAPSHOT'
+    STORAGE_VERSION = "0.7.1-SNAPSHOT"
```
Comment (armiol) [inline build.gradle]: Should be `GAE_JAVA_VERSION` instead, after the library name.

File: gcd/src/main/java/org/spine3/server/storage/datastore/Entities.java [hunk 2]
```diff
@@ -51,13 +50,14 @@
...
     private static final String VALUE_PROPERTY_NAME = "value";
+    private static final String DEFAULT_MESSAGE_FACTORY_METHOD_NAME = "getDefaultInstance";
 
     private Entities() {
     }
 
     /**
      * Retrieves a message of given type, assignable from {@code Message}, from an {@link Entity}.
-     *
+     * <p>
```
Comment (armiol) [inline gcd/src/main/java/org/spine3/server/storage/datastore/Entities.java]: Why do we need empty paragraphs? Here and below.

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov please see my comments.

**Comment by dmdashenkov** [comment]
@armiol Fixed your comments. PTAL again

**Review by armiol — APPROVED** [review]

**Pull request merged**